src/js/renderers/dash.js

```diff
@@ -7,6 +7,9 @@ import {renderer} from '../core/renderer';
 import {createEvent} from '../utils/general';
 import {typeChecks} from '../utils/media';
 import {HAS_MSE} from '../utils/constants';
+import {loadScript} from '../utils/dom';
+
+let scriptPromise;
```

Comment:
Is this used to cache the script loading? Meaning, if the code gets inserted once and then we load another renderer, and we try to call the previous renderer, will this help?